test/JDBC/upgrade/14_7/schedule

```diff
@@ -367,3 +366,4 @@ app_name
 case_insensitive_collation
 sys-has_perms_by_name
 sys-has_perms_by_name-dep
+bbf_view_def_upgrade-before-15_2
```

Comment:
This should be `bbf_view_def` (let's also remove "upgrade" from the file name).

Reply:
Sure